readme.md

```diff
@@ -75,6 +75,7 @@ The term was coined by Eric Evans in his book of the same title.
 
 - [The Anatomy of Domain-Driven Design - The Infographic](https://leanpub.com/theanatomyofdomain-drivendesign) - An infographic on the anatomy of Domain-Driven Design.
 - [Architecting Modern Web Applications with ASP.NET Core and Microsoft Azure](https://dotnet.microsoft.com/download/thank-you/aspnet-ebook) - Provides end-to-end guidance on building monolithic web applications using DDD, ASP.NET Core, and Azure. [eShopOnWeb sample](https://github.com/dotnet-architecture/eShopOnWeb)
+- [Architecture Patterns with Python](https://www.cosmicpython.com/book/preface.html) - A Book about Pythonic Application Architecture Patterns for Managing Complexity.
```

Comment:
Mind making the casing consistent with the other descriptions?  `A book about pythonic application architecture patterns for managing complexity.`

Reply:
Sure